Xamarin.Forms.Core/NavigationMenu.cs

```diff
@@ -64,6 +64,8 @@ public void Remove(Page target)
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
 
+		void INavigationMenuController.SendTargetSelected(Page target) => SendTargetSelected(target);
+
 		internal void SendTargetSelected(Page target)
```

Comment:
why are we keeping the internal one when we now have a public `SendTargetSelected()` ?